sharedmodels/utils/uri.py

```diff
@@ -0,0 +1,50 @@
+"""
```

Comment:
Did you mean to add this new file? Are URI utils still in `csvqb` as well?

Reply:
Yes. The uri utils need to be here because sharedmodels was accessing csvqb utilities via a cross-package import which doesn't agree with pylance/pyrite (delete as appropriate.)